package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@less/root",
+  "workspaces": ["packages/*"], 
```

Comment:
No.3 lerna v7 [deprecated `bootstrap`](https://lerna.js.org/docs/legacy-package-management#migrating-from-lerna-bootstrap-lerna-add-and-lerna-link-in-lerna-v7-and-later) command, we should use standalone tool to install and symlink npm packages.